.github/workflows/release_staging.yml

```diff
@@ -0,0 +1,68 @@
+name: Release staging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release_staging:
+    runs-on: macos-12
+    environment: Staging
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Checkout the project
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Cache bundle gems
+      id: bundlerCache
+      uses: actions/cache@v3
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+
+    - name: Bootstrap the project
+      run: make bootstrap
+
+    - name: Install the Apple certificate and provisioning profile
```

Comment:
What do you think about using Fastlane Match? 🤔

Reply:
@markgravity 
`fastlane match` is a tool that helps sync certificates and profiles across the development team. So every developer doesn't have to manually set up, and update certificates, and profiles on his/her local machine. It removes the tedious work and even saves us from troubles, and issues with code signing.

However, there are two points:
- This project basically is implemented by only one developer (that's me 😅 for that matters)
- To set up `fastlane match` we need to create a **separate** and **private** repo to store code signing identities. It makes things a little more complicated than necessary. Also, it's a little bit weird that this repo is public for education and reference purposes but it depends on another private repo IMHO (There may be a lot of public projects that go with this approach though)

To conclude, I think using `fastlane match` does not have many benefits and is not worth adding to this project 🙏 
What do you think?

Reply:
@Thieurom All good for me 👍